src/main/java/org/gridsuite/study/server/service/GeoDataService.java

```diff
@@ -38,29 +41,37 @@ public GeoDataService(@Value("${backing-services.geo-data.base-uri:http://geo-da
         this.geoDataServerBaseUri = geoDataServerBaseUri;
     }
 
-    public String getLinesGraphics(UUID networkUuid, String variantId) {
+    public String getLinesGraphics(UUID networkUuid, String variantId, List<String> linesIds) {
         var uriComponentsBuilder = UriComponentsBuilder.fromPath(DELIMITER + GEO_DATA_API_VERSION + "/lines")
             .queryParam(NETWORK_UUID, networkUuid);
 
         if (!StringUtils.isBlank(variantId)) {
             uriComponentsBuilder.queryParam(QUERY_PARAM_VARIANT_ID, variantId);
         }
 
+        if (linesIds != null) {
```

Comment:
We don't manage Country Query Parameter anywhere ?

Reply:
No. It's implemented in geo-data-server but not used yet.